### PR TITLE
adding counjt down timer to the home page

### PR DESCRIPTION
**Pull request by magierowskiTombola**
-added the count down timer
-deleted unused JQuery files.

### CODE REVIEW

File: Tombolathon/Tombolathon/Content/Site.css [hunk 1]
```diff
@@ -110,4 +110,12 @@ tr{
     tr:hover {background-color: #f5f5f5}
 }
 
-tr:nth-child(even) {background-color: #f2f2f2}
\ No newline at end of file
+tr:nth-child(even) {background-color: #f2f2f2}
+
+#getting-started
```
Comment (LukeHillTombola) [inline Tombolathon/Tombolathon/Content/Site.css]: I'd keep Site.css as minimalist and 'backbone of the site' as possible. Apply specific styling in separate css files that need to override behaviours. Really, only core html elements should be here. Classes and ids should live separately.

File: Tombolathon/Tombolathon/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -28,6 +28,10 @@
 
 </div>
 
+<div class="row">
+    <div id="getting-started"></div>
```
Comment (LukeHillTombola) [inline Tombolathon/Tombolathon/Views/Home/Index.cshtml]: Seems like a strange name for the countdown timer.

File: Tombolathon/Tombolathon/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -90,8 +94,15 @@
 @section Scripts {
 
     <script type="text/javascript" src="~/Scripts/loadgo.js"></script>
-
-    <script type="text/javascript">
+    <script type="text/javascript" src="~/Scripts/Countdown/jquery.countdown.js"></script>
```
Comment (LukeHillTombola) [inline Tombolathon/Tombolathon/Views/Home/Index.cshtml]: We can make this a self closing tag.

Not super important - just a note that self closing tags exist if you didn't know about 'em.

Comment (magierowskiTombola) [inline Tombolathon/Tombolathon/Views/Home/Index.cshtml]: seems like making it self closing breaks the code. (the timer no loner shows then)

**Comment by LukeHillTombola** [comment]
:shipit:

**Pull request merged**